lib/internal/modules/esm/loader.js

```diff
@@ -378,10 +378,6 @@ class ESMLoader {
         );
       }
     }
-
-    // [1] ensure hook function is not bound to ESMLoader instance
```

Comment:
Because otherwise, since `addCustomLoaders` is now called multiple times (once for each loader), it would lead to `preload` being called multiple times, which in turn would execute the global preload hook multiple times, which would crash (you can try reverting this change; a test crashes).

Reply:
The comment wasn't related to the `this.preload()` call, but to the `FunctionPrototypeBind` above. Anyway, it's been removed from `main` so there's no problem (except a git conflict to resolve).

Reply:
Ah! I think I didn't realize this comment was a footnote of the previous lines and must have accidentally removed it 🤔 will rebase